spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb

```diff
@@ -15,7 +17,25 @@
     let(:repos)              { Dir.glob(File.join(repo_dir, "*")) }
     let(:repo_dir_structure) { %w[hello_world.tf] }
 
+    let(:hello_world_vars_response) do
+      JSON.parse(File.read(File.join(__dir__, "../../../../../lib/terraform/runner/data/responses/hello-world-variables-success.json")))
```

Comment:
You can also use the engine.root to get a path relative to the top of the plugin
```suggestion
      JSON.parse(ManageIQ::Providers::EmbeddedTerraform::Engine.root.join("spec/lib/terraform/runner/data/responses/hello-world-variables-success.json").read)
```